src/io-operations.js

```diff
@@ -57,6 +57,12 @@ function useDefaultIOMethod() {
   writeManifest = defaultIOMethod.writeManifest;
 }
 
+function getCredentialsFromEnv() {
```

Comment:
A bit nitpicky, but this function isn't needed. Let's replace with the following code inserted before line 54:

```js
if (process.env.IMD_USERNAME && process.env.IMD_PASSWORD) {
  username = process.env.IMD_USERNAME;
  password = process.env.IMD_PASSWORD;
}
```

^ More straightforward - no need to wrap the values in a function that returns an object

Reply:
Can do. I wasn't sure how comfortable you'd be with in-lining the work like that, but I suppose it's simple enough that there's no gain from pulling it out into a function.